Sources/ComposableArchitecture/SharedState/References/ValueReference.swift

```diff
@@ -114,18 +114,20 @@
     #endif
 
     init(
-      initialValue: @autoclosure @escaping () -> Value,
+      initialValue: Value,
```

Comment:
I think we were being a little too precious here with the autoclosure, which needed to be escaped. I think simply passing the value along is probably better.